engine/admin-api/domain/entity/version.go

```
@@ -45,8 +50,18 @@ const (
 	VersionStatusCritical  VersionStatus = "CRITICAL"
 )
 
-func (e VersionStatus) String() string {
-	return string(e)
+func (vs VersionStatus) String() string {
+	return string(vs)
+}
+
+func (vs VersionStatus) Validate() bool {
```

IMO we should return an error here like we are doing in the rest of Validate funcs

---

I see, I didn't think of it then, I agree returning an error is best